packages/react-router/index.js

```diff
@@ -489,6 +489,8 @@ export function useRoutes(routes, basename = '', caseSensitive = false) {
 
   let navigate = useNavigate();
   let location = useLocation();
+  let locationPreloadRef = useRef();
```

Comment:
```suggestion
  let locationPreloadRef = React.useRef();
```

Just a nit pick, but the rest of the library uses the default import, so this breaks the mold otherwise. If you want to switch to named imports, I'd do that in a separate PR.